docs/modules/reference/pages/glossary.adoc

```diff
@@ -12,18 +12,32 @@ ALEC groups related alarms so you can better manage the more complex situations
 Application Perspective Monitoring (APM):: A method that uses the Minion infrastructure to monitor a service’s availability from different perspectives to help identify if nodes are down for everyone or if an outage affects only certain sites. When a service is not responsive an outage will be generated with the corresponding perspective that sees the outage.
 (See xref:operation:application-perspective-monitoring/introduction.adoc#application-perspective[Application Perspective Monitoring].)
 
+Azure:: Microsoft Azure is a cloud-based hosting service that allows users to launch and maintain Microsoft Windows or Linux virtual machines.
+
 Cassandra:: Open-source distributed/scale-out NoSQL database designed to hold large amounts of structured or unstructured data across many servers.
 (See xref:deployment:time-series-storage/newts/introduction.adoc#ga-opennms-operation-newts[Newts].)
 
+CircleCI:: CircleCI is a continuous integration and delivery platform that hosts OpenNMS's CI/CD environment. It integrates with and monitors OpenNMS GitHub repositories, and automatically launches and tests a new build when a commit is detected.
+
+Discourse:: Discourse is an open-source forum platform built on the Ember.js framework. It forms the basis for OpenNMS's community forums (https://opennms.discourse.group/).
```

Comment:
```suggestion
Discourse:: Discourse is an open-source forum platform that hosts OpenNMS's community forums (https://opennms.discourse.group/).

```
For this one, I would remove the platform information and maybe combine the two sentences, and add a second sentence about what people can expect: knowledge base articles, support, announcements, and general discussion about all things related to OpenNMS.